.editorconfig

```diff
@@ -1,10 +1,33 @@
+# https://editorconfig.org
+
 root = true
 
 [*]
+charset = utf-8
 end_of_line = lf
+indent_size = 4
+indent_style = tab
```

Comment:
Why not spaces ?

Reply:
I don't know, is the pattern used by Nextcloud.

To PHP files is a bit different from PSR.

Maybe because is oldest of PSR.

Nextcloud is a project since 2010. The PSR that defined spaces was made in 2012

Reply:
I prefer to follow PSR but is very strange to have only LibreSign following a different pattern of Nextcloud. Because this we follow the pattern from server and most of apps.

Reply:
https://github.com/nextcloud/coding-standard

Reply:
From PSR we should use spaces over tabs: https://www.php-fig.org/psr/psr-2/#1-overview

Also, if we are thinking about to have LibreSign as a product without all NextCloud actual things make sense to have different code standard or we can just follow it.

Reply:
Isn't possible to be a product that isn't a Nextcloud app.

To to this will be a very very big refactor, we will need to replace the backend and frontend framework of LibreSign and implement a lot of functionalities that Nextcloud solve to us.

For now we will follow the Nextcloud pattern to be easy to other Nextcloud developers.